app/styles/drawing-tool.scss

```diff
@@ -245,7 +252,8 @@ $text: Arial, sans-serif;
       position: absolute;
       display: block;
       content: ' ';
-      background-image: url('../assets/menu-secondary-caret.svg');
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='40' height='40' viewBox='0 0 40 40'%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cg%3E%3Cg%3E%3Cpath d='M0 0H40V40H0z' transform='translate(-1947.000000, -892.000000) translate(1947.000000, 892.000000)'/%3E%3Cpath fill='%230481A0' d='M7 0L7 7 0 7z' transform='translate(-1947.000000, -892.000000) translate(1947.000000, 892.000000) translate(28.500000, 29.500000)'/%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/svg%3E%0A");
+      // background-image: url('../assets/menu-secondary-caret.svg');
```

Comment:
>Switched background image of submenus to use data uri instead of an svg import so that it is not mangled when question-interactives applies the react svg plugin in webpack

Assuming this is the problem of the `@svgr/webpack` loader modifying SVGs in ways that cause them not to function as expected, this is handled in CLUE and DESE with some additional webpack configuration in the `webpack.config.js`:
```javascript
        {
          test: /\.svg$/i,
          exclude: /\.nosvgo\.svg$/i,
          oneOf: [
            {
              // Do not apply SVGR import in CSS files.
              issuer: /\.(css|scss|less)$/,
              type: 'asset'
            },
            {
              issuer: /\.tsx?$/,
              loader: '@svgr/webpack',
              options: {
                svgoConfig: {
                  plugins: [
                    // leave <line>s, <rect>s and <circle>s alone
                    // https://github.com/svg/svgo/blob/master/plugins/convertShapeToPath.js
                    { convertShapeToPath: false },
                    // don't minify "id"s (i.e. turn randomly-generated unique ids into "a", "b", ...)
                    // https://github.com/svg/svgo/blob/master/plugins/cleanupIDs.js
                    { cleanupIDs: { minify: false } },
                    // don't add random prefixes to "class"es and "id"s
                    // https://github.com/svg/svgo/blob/master/plugins/prefixIds.js
                    { prefixIds: false },
                    // leave "stroke"s and "fill"s alone
                    // https://github.com/svg/svgo/blob/master/plugins/removeUnknownsAndDefaults.js
                    { removeUnknownsAndDefaults: { defaultAttrs: false } }
                  ]
                }
              }
            }
          ]
        }

```
This configures the SVGO part of the `@svgr/webpack` loader to
1. always disable certain optimizations that I/we have found to be destructive.
2. provide an opt-out for files named `.nosvgo.svg` which disables optimization altogether in cases where disabling the previous optimizations is insufficient.

Reply:
@kswenson I tried a variant of that but the other issue is that I would need to update the `drawing-tool` webpack to copy the one svg used in the .css file to the `dist` folder so that `question-interactives` webpack could find it when it is referenced in the TypeScript code.  I thought it was more clean to use a data url.